server/svix-server/src/core/types.rs

```diff
@@ -471,7 +471,7 @@ impl EndpointSecretMarker {
         let encrypted = (v & Self::ENCRYPTED_FLAG) != 0;
         let v = v & !Self::ENCRYPTED_FLAG;
         let type_ = EndpointSecretType::try_from(v)
-            .map_err(|_| crate::error::Error::Generic("Invalid marker value".to_string()))?;
+            .map_err(|_| crate::err_generic!("Invalid marker value"))?;
```

Comment:
Is there a reason not to `use crate::err_generic` and get rid of the prefix? I imagine that macro name is pretty unique. (Also lines 545 and 609)

Reply:
I think having some prefix is nice. Because we'll also have `database!` which is less unique...

Reply:
I didn't have a specific reason to use the `crate::`, other than it's what was already there. Removed.